src/bps/user.py

```diff
@@ -68,3 +68,53 @@ def user_password():
         return { "auth": auth }
     except BaseException as e:
         return error_resp(e)
+
+@bp.route('/state', methods=['PUT'])
+def user_set_state():
+    auth   = request.headers.get("Authorization")
+    params = request.json
+
+    user = check_auth(auth)
+    if user == None:
+        abort(403)
+
+    try:
+        if( 'id'    in params and
+            'state' in params
+            ):
+            id = params['id']
```

Comment:
em vez de passar o id por parametro, pega do user
```suggestion
            id = user['id']
```